Source/Rendering/SvgRenderer.cs

```diff
@@ -135,7 +135,7 @@ private static Graphics CreateGraphics(Image image)
             g.CompositingQuality = CompositingQuality.HighQuality;
             g.TextRenderingHint = TextRenderingHint.AntiAlias;
             g.TextContrast = 1;
-            g.InterpolationMode = InterpolationMode.NearestNeighbor;
```

Comment:
@paulushub 

Do you need to change this setting ?
I recommend that you change this setting carefully, such as by creating another PR.

Reply:
It was added by a PR we allowed to record the contribution and close the issue as we planned to improve the implementation.
However, as you can see it is causing the build to fail with a SVG-Logo. 
So, I set it to `Default` to make all the tests pass as before.